pydy/tests/test_bodies.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+
+SYMPY_VERSION = sm.__version__
+
+
+class TestBodies():
+
+    def test_body_init(self):
+        body = Body('body')
+
+        assert body.name is None
+        assert body.mass == Symbol('mass')
```

Comment:
You can look here for the correct implementation https://github.com/pydy/pydy/pull/249/files#diff-5727bd04fc961cdc0987b7eba64b51a7R449